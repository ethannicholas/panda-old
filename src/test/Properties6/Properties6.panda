class Test {
    @readonly
    property name := "Hello, my name is {}!"

    @class
    method main() {
        def test := Test()
        Console.writeLine(test.name)
        test.name := "This won't compile!"
    }
}