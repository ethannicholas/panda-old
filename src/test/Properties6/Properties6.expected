error: Properties6.panda:9:19: cannot assign to 'test.name()'
1 error
